GitUI/Editor/FileViewer.cs

```diff
@@ -1367,7 +1387,6 @@ private void NextChangeButtonClick(object sender, EventArgs e)
                 {
                     if (emptyLineCheck)
                     {
-                        internalFileViewer.FirstVisibleLine = Math.Max(line - 4, 0);
                         internalFileViewer.LineAtCaret = line;
```

Comment:
Have you removed this because you are sure that setting `LineAtCaret` makes this line visible?